### PR TITLE
Make Store examples' syntax match their code

### PR DESCRIPTION
**Pull request by alexcjohnson**
... by extending `load_example` to ignore lines with "# no-run" comments. Stores are "special" components that can't (at least for now) be added outside the original layout, so we create them in the base `run.py`, and we have to strip them out of the examples before running them within the greater docs app. Would be nice to remove that restriction, but until then this at least keeps the two pieces in sync.

This is an updated version of #496 by @StefanTippelt 

Post-merge checklist:

The master branch is auto-deployed to `dash.plot.ly`.
Once you have merged your PR, wait 5-10 minutes and check dash.plot.ly 
to verify that your changes have been made.

- [X] I understand

### CODE REVIEW

File: tutorial/examples/core_components/store_clicks.py [hunk 1]
```diff
@@ -11,8 +11,15 @@
 app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
 
 app.layout = html.Div([
-
-
+    # The memory store will always get the default on page refreshes
```
Comment (byronz) [inline tutorial/examples/core_components/store_clicks.py]: page refresh?

File: tutorial/tools.py [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
     def wrapper(path):
         try:
             return func(path)
         except Exception as e:
-            print('\nError running {}\n{}'.format(
-                path,
-                ('======================================' +
-                 '======================================')
-            ))
+            print('\nError running {}\n{}'.format(path, '=' * 76))
```
Comment (byronz) [inline tutorial/tools.py]: 👍

File: tutorial/tools.py [hunk 3]
```diff
@@ -51,12 +50,26 @@ def load_example(path):
             'print("Running")\n    # app.run_server'
         )
 
+        # if there are lines that should be included in the syntax but
+        # not executed, simply put this comment at the end of the line.
+        no_run = '# no-run'
```
Comment (byronz) [inline tutorial/tools.py]: Shall we use not-exec or no-py-exec? I didn't get it in the comments until here

Comment (alexcjohnson) [reply tutorial/tools.py]: sure, I can change it to `no-exec` if you think that's clearer.

**Comment by alexcjohnson** [comment]
Buttons moved into the table:
Screen Shot 2019-06-11 at 9 53 16 PM

**Pull request merged**